Makefile

```diff
@@ -84,13 +84,6 @@ export SWARM_STACK_NAME_NO_HYPHEN = $(subst -,_,$(SWARM_STACK_NAME))
 export DOCKER_IMAGE_TAG ?= latest
 export DOCKER_REGISTRY  ?= itisfoundation
 
-# NOTE: this is only for WSL1 as /etc/hostname is not accessible there
-ifeq ($(IS_WSL),WSL)
-ETC_HOSTNAME = $(CURDIR)/.fake_hostname_file
-export ETC_HOSTNAME
-host := $(shell echo $$(hostname) > $(ETC_HOSTNAME))
-endif
-
```

Comment:
Is this not used anymore? I remember I had to use this at some point. Not because I was using WSL though. So I had to comment out line 87 and 92. I don't remember anymore why I had to do that, but could it be useful to keep this somewhere handy?

Reply:
Makes sense. Thanks for the cleanup